### PR TITLE
860 Added Navbar, Hero Section, and Footer for Non-Root Pages

### PR DESCRIPTION
**Pull request by Aaryanpal**
# 🔗 Issue
#860 

# ✍️ Description
Revamp the Design for Navbar, Hero Section and Footer for Non-Root Pages

# 📷 Screenshots/Demos

[Screencast from 2024-08-30 18-06-07.webm](https://github.com/user-attachments/assets/3f0408c5-f8ab-495c-8db1-1e3093ff144a)

### CODE REVIEW

**Comment by Aaryanpal** [comment]
@kasugaijin These changes are optimized for screens larger than 1440px. If you can provide the designs for tablet and mobile, I can make it fully responsive. The Figma file has some designs, but they seem incomplete. Could you guide me through it?

**Comment by kasugaijin** [comment]
@Aaryanpal for desktop please build to this spec https://www.figma.com/design/0jVgYASUJy0KiX3BVc3dFM/Tasks?node-id=1150-254&t=togPhk8uYLJsyAeU-1
for mobile please build to this spec https://www.figma.com/design/0jVgYASUJy0KiX3BVc3dFM/Tasks?node-id=1318-489&t=togPhk8uYLJsyAeU-1

I cannot remember what breakpoints we use, but it's all with bootstrap, and you can check some of the other pages to see what classes we use to switch from desktop to mobile.

**Comment by Aaryanpal** [comment]
Thanks for explaining @kasugaijin 😊

File: config/locales/en.yml [hunk 1]
```diff
@@ -78,29 +78,39 @@ en:
...
-      no_download: "No dowload needed"
-      manage: "Manage operations in one place"
-      save: "Save time and spend it on what matters"
-      btn_title: "Request an account"
-      subtitle: "Manage your operations"
-      subtitle_highlight: "in one place"
-      subtitle_description: "No more spreadsheets, no more paper, no more headaches."
+      title: "Manage"
+      pet: "Pets"
+      description: "Bring your operations to the digital age. Handle everything from staffing, foster scheduling, and adoptions, all from one free app."
```
Comment (kasugaijin) [inline config/locales/en.yml:83]: could we change the word `app` to `website`? After seeing this on my screen, alive, I don't want to give people the impression this is a mobile app. So can we change this for all cases below as well?

File: app/views/root/index.html.erb [hunk 2]
```diff
@@ -1,162 +1,315 @@
...
+                <div class="col-auto">
+                <%= image_tag "icon_6.png", alt: "Tree icon", class: "img-fluid", style: "max-width: 75px;" %>
+                </div>
+                <div class="col-auto">
+                <%= image_tag "icon_7.png", alt: "Tool Library", class: "img-fluid", style: "max-width: 75px;" %>
+                </div>
+            </div>
+        </div>
+        <div class = "mt-5 column d-flex flex-column align-items-center justify-content-center">
+            <a href="#!" class="btn btn-outline-secondary px-8 mb-2 text-primary"><%= t('.learn_more_about')%></a>
```
Comment (kasugaijin) [inline app/views/root/index.html.erb:312]: Can we `link_to` to https://rubyforgood.org/ here please

**Comment by kasugaijin** [comment]
@Aaryanpal looking really good! Just a couple small comments. Did you want to add the mobile design fixes in this PR or shall we separate that out?

File: config/locales/en.yml (hunk 1 above)
Comment (Aaryanpal) [reply config/locales/en.yml:83]: sure. Will replace this as mention

**Comment by Aaryanpal** [comment]
> @Aaryanpal looking really good! Just a couple small comments. Did you want to add the mobile design fixes in this PR or shall we separate that out?

I'll try to bring it all together in one piece.

**Comment by kasugaijin** [comment]
@Aaryanpal awesome thank you! I will be off grid this weekend, but back on Monday to review stuff if you are ready for it. 

Thanks for your hard work!

**Comment by Aaryanpal** [comment]
- [x] Mobile  ScreenCast

[Screencast from 2024-08-31 23-27-15.webm](https://github.com/user-attachments/assets/7dacb6ff-8e2d-4369-9575-83857935ea21)

- [x] Laptop and Wider Screen  ScreenCast

[Screencast from 2024-08-31 23-28-57.webm](https://github.com/user-attachments/assets/3a355533-6ac3-49b4-8054-60b7c8da80a4)

**Comment by kasugaijin** [comment]
@Aaryanpal thank you. This is a solid start for the org root page.

**Review by kasugaijin — APPROVED** [review]
LGTM

**Pull request merged**